test/Microsoft.Azure.Devices.Edge.Test.Common/linux/PackageManagement.cs

```diff
@@ -0,0 +1,285 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common.Linux
+{
+    using System;
+    using System.IO;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Serilog;
+
+    public class PackageManagement
+    {
+        Lazy<Task<LinuxStandardBase>> lsbTask =
+            new Lazy<Task<LinuxStandardBase>>(
+                async () =>
+                {
+                    var cts = new CancellationTokenSource();
+                    cts.CancelAfter(10000);
```

Comment:
We need to use the token generated in `SetupFixture`. Which means it needs to be passed to the constructor. Then you'll initialize `lsbTask` in the constructor instead of here.

Reply:
there there.